packages/suite/src/components/suite/Banners/MessageSystemBanner.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+
+import * as routerActions from '@suite-actions/routerActions';
+import * as messageSystemActions from '@suite-actions/messageSystemActions';
+import { useActions } from '@suite-hooks';
+import Wrapper from './components/Wrapper';
+
+import type { Message } from '@suite-types/messageSystem';
+
+type Props = {
+    message: Message;
+};
+
+const MessageSystemBanner = ({ message }: Props) => {
+    const { cta, variant, id, content, dismissible } = message;
+
+    const { goto, dismissNotification } = useActions({
+        goto: routerActions.goto,
+        dismissNotification: messageSystemActions.dismissMessage,
+    });
+
+    const getActionConfig = () => {
```

Comment:
Maybe worth memoizing here? Same for `getDismissalConfig`.

Reply:
All three functions memoized. I checked the code and did not find anything else to memorize 👍